app/build.gradle

```diff
@@ -25,6 +25,15 @@ android {
     }
 }
 
+ext {
+    live_data_version = '1.0.0-alpha1'
+    anko_version = '0.10.0'
+    rxjava_version = '2.1.0'
+    retrofit_version = '2.0.2'
+    mockito_version = '1.10.19'
```

Comment:
Can you upgrade this to Mockito 2, because I believe now we can finally mock final classes. https://stackoverflow.com/questions/14292863/how-to-mock-a-final-class-with-mockito/40018295#40018295...